docs/clients.md

```diff
@@ -129,28 +129,33 @@ The official Jellyfin Kodi plugin.
 
 #### Installing
 
-1. Download the add-on as a [zip file](https://github.com/jellyfin/jellyfin-kodi/archive/master.zip)
-    * It will be saved as `jellyfin-kodi-master.zip`
-2. Install Jellyfin for Kodi
+1. Download the repository installer as a [zip file](https://repo.jellyfin.org/releases/client/kodi/repository.jellyfin.kodi.zip)
+    * It will be saved as `repository.jellyfin.kodi.zip`
+2. Install the Jellyfin repository
```

Comment:
Might be good to specify that these steps are done from within Kodi (not Jellyfin)

Reply:
Made it a bit clearer to see that the installing is done on the kodi/client side, not the server side.